src/mate/src/Command/DebugCapabilitiesCommand.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Mate\Command;
+
+use Psr\Log\LoggerInterface;
+use Symfony\AI\Mate\Discovery\CapabilityCollector;
+use Symfony\AI\Mate\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Display all MCP capabilities grouped by extension.
+ *
+ * @author Johannes Wachter <johannes@sulu.io>
+ */
+class DebugCapabilitiesCommand extends Command
+{
+    private CapabilityCollector $collector;
+
+    public function __construct(
+        LoggerInterface $logger,
+        private ContainerBuilder $container,
+    ) {
+        parent::__construct(self::getDefaultName());
+
+        $rootDir = $container->getParameter('mate.root_dir');
+        \assert(\is_string($rootDir));
+
+        $this->collector = new CapabilityCollector($logger, $rootDir);
```

Comment:
What if we use dependency injection instead?

Reply:
will do that in an own PR later on